crates/sdk/src/tx.rs

```diff
@@ -2948,7 +2939,7 @@ pub async fn build_transfer<N: Namada>(
     // This has no side-effect because transaction is to self.
     let (transparent_amount, transparent_token) =
         if source == masp_addr && target == masp_addr {
-            // TODO Refactor me, we shouldn't rely on any specific token here.
+            // TODO: Refactor me, we shouldn't rely on any specific token here.
```

Comment:
we could link this to https://github.com/anoma/namada/issues/1677

Reply:
thx, I wasn't sure which issue to link this to